### PR TITLE
samples/littlefs: Fix rcar boards selecting Flash API

### PR DESCRIPTION
**Pull request by de-nordic**
The rcar boards use block variant of LittleFS and Disk Access API, but have been using proj.conf for Flash API based access. The commit adds section building boards with pro_blk.conf into sample.yaml, where rcar boards will now be allowed, and removes redundant configuration lines from board specific conf files.

### CODE REVIEW

File: samples/subsys/fs/littlefs/boards/rcar_h3ulcb_r8a77951_a57.conf [hunk 1]
```diff
@@ -6,12 +6,3 @@
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=8192
 CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
```
Comment (xakep-amatop) [inline samples/subsys/fs/littlefs/boards/rcar_h3ulcb_r8a77951_a57.conf:8]: these configs aren't applied at all with build command:
```
west build -p always -b rcar_h3ulcb/r8a77951/a57 samples/subsys/fs/littlefs -- -DCONF_FILE=prj_blk.conf
```

**Comment by xakep-amatop** [comment]
Review finished.

**Comment by de-nordic** [comment]
> Review finished.

I have reworked configurations for your boards. Now they also enabled MMC and disable SDMCC Disk Access Drivers, the diff now, for , looks like this:
image

**Review by xakep-amatop — APPROVED** [review]

**Comment by xakep-amatop** [comment]
Checked changes, everything works nicely.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**